proto/vulcanize/nameservice/v1beta1/query.proto

```diff
@@ -0,0 +1,211 @@
+syntax = "proto3";
+package vulcanize.nameservice.v1beta1;
+
+import "vulcanize/nameservice/v1beta1/nameservice.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/tharsis/ethermint/x/nameservice/types";
+
+// Query defines the gRPC querier service for nameservice module
+service Query {
+  // Params queries the nameservice module params.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/ethermint/nameservice/v1/params";
+  }
+  // List records
+  rpc ListRecords(QueryListRecordsRequest) returns (QueryListRecordsResponse){
```

Comment:
Let's include test CLI runs for all of these queries.

Reply:
I am writing those tests on separate branch